### PR TITLE
Integration v02 

### PR DESCRIPTION
**Pull request by Atouzani7**
Adding 
- autoplay video 
- random 
- count 
- timer

### CODE REVIEW

**Review by guenk — APPROVED** [review]
NO evil

File: frontend/src/components/Timer/Timer.jsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from "react";
```
Comment (LoicBrassart) [inline frontend/src/components/Timer/Timer.jsx]: Stop important React

File: frontend/src/components/Toastify/Toast.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
```
Comment (LoicBrassart) [inline frontend/src/components/Toastify/Toast.jsx]: Stop important React

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Please remove useless React imports

**Review by guenk — APPROVED** [review]
well done

**Review by LoicBrassart — APPROVED** [review]
Saw no Evil

**Pull request merged**